modin/experimental/xgboost/xgboost.py

```diff
@@ -25,7 +25,7 @@
 LOGGER = logging.getLogger("[modin.xgboost]")
 
 
-class DMatrix(xgb.DMatrix):
```

Comment:
@prutskov why don't you inherit anymore?

Reply:
Inheritance doesn't include any benefits, because Modin DMatrix provided very limited set of supported properties/methods. Also, inheritance includes additional problems for `doc_checker` for `labels` parameter of `__init__` dunder method.

Reply:
ок